bool BallDetected
float64 currentRadius


